### ISSUE TITLE
NTR: Ovarian biopsy: cystadenomas  (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9148]
New term request Ovarian biopsy: cystadenomas  (elthat_100322014253): Ovary

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000672
- label: Ovarian cystadenoma
- parents: [Ovarian neoplasm (HP:0100615)](https://hpo.jax.org/app/browse/term/HP:0100615)
- pmid: PMID:30725635
- def: A common, benign ovarian tumor that carries a good prognosis. The two most frequent types of cystadenomas are serous and mucinous cystadenomas whereas endometrioid and clear cell cystadenomas are rare.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9148
- synonyms:
